cmd/gh2gcs/README.md

```diff
@@ -34,3 +34,16 @@ INFO Writing assets to /tmp/test/kubernetes/kubernetes/v1.18.0
 ...
 
 ```
+
+## Use case
+
+GitHub has a rate limit on downloading release artifacts. The artifacts that are often used (e.g. in CI) are uploaded to Google Cloud Storage, as GCS doesn't have read rate limits. This improves test stability as flakes related to hitting rate limits are avoided.
+
+## SIG Release managed buckets
+
+The following GCS buckets are managed by SIG Release:
+
+- k8s-artifacts-cni - contains CNI plugins artifacts
```

Comment:
Let's link the CNI plugins repo here, too.

Reply:
@saschagrunert Do we have our repo or linking to https://github.com/containernetworking/plugins is okay?

Reply:
I think linking to https://github.com/containernetworking/plugins should be fine :) We could also recommend to the manager to watch those repositories for possible new releases.

Reply:
@saschagrunert Fixed, PTAL.